app/models/goods_nomenclature.rb

```diff
@@ -51,4 +51,14 @@ def id
   def code
     goods_nomenclature_item_id
   end
+
+  def rules_of_origin(*args, **kwargs)
+    return nil unless try(:declarable?)
```

Comment:
Don't we always have this answer?

Reply:
Not on `Chapter` which also inherits from `GoodsNomenclature`